src/main/java/com/openlattice/hazelcast/HazelcastMap.java

```diff
@@ -67,7 +67,7 @@
     LINKING_EDGES,
     LINKING_ENTITIES,
     LINKING_ENTITY_VERTICES,
-    LINKING_FEEDBACKS,
+    LINKING_FEEDBACK,
```

Comment:
Is there a mpastore for this? If there is the name needs to be updated there too

Reply:
+1

Reply:
`LinkingFeedbackMapstore`

Reply:
ok done... I think...  Is the commit below enough?

Reply:
If you're using intellij to do the refactor it should update all usages for you so you don't have to search through all the projects or worry about missing any (and you can also find all usages using intellij to double check)